docs/front-end/standaarden/digitoegankelijk/run-axe.md

```diff
@@ -50,15 +70,19 @@ jobs:
               "txt": "text/plain",
               "xml": "text/xml"
             }
-          # De directory waar de bestanden staan. Zet dit op de output folder van
-          # de gebouwde bestanden als er een build step in het project zit.
-          directory: frontend/ 
-        # Het daadwerkelijk runnen van Axe. Hier wijst het naar `localhost:8080` wat
-        # de files servert middels de HTTP server. Hier is `/index.html` geopend, maar
-        # dit kan je veranderen door elke andere pagina die je wilt gebruiken.
-        #
-        # Tevens worden de AA WCAG guidelines gebruikt. Voor striktere checks kan ook
-        # de "wcag2aaa" tag worden gebruikt. De volledige lijst is beschikbaar op:
-        # https://www.deque.com/axe/core-documentation/api-documentation/#axecore-tags
-      - run: axe http://localhost:8080/index.html --exit --tags wcag2aa
+          port: 3000
+          directory: build/
+          index-files: |
+            ["index.html"]
+
+      - name: Run Axe to check for WCAG compliance
+        run: pnpm run lint:wcag
+
 ```
+
+## NPM script
+
+Het is raadzaam om ook direct een npm script in te richten zodat je Axe ook lokaal kan draaien. **Let op**: onderstaand script vereist dat je lokaal een webserver draait op: `http://localhost:3000`.
+```js
+"lint:wcag": "axe http://localhost:3000/index.html, http://localhost:3000/blog --exit --tags wcag2a"
```

Comment:
Ik bedacht me later nog: A is minder dan AA. Als je de strengste eisen wilt stellen, dan moet je `wcag2aaa` gebruiken. Zie https://www.w3.org/WAI/WCAG2A-Conformance

Reply:
Check. Ik pas hem aan naar AA.

Reply:
@MrSkippy ok verder zo?